pkgs/tools/nix/nixel/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, testVersion
```

Comment:
nix-review complains about: `Function called without required argument "testVersion"`

Reply:
@Lassulus Good catch. This PR was open for a long time, and `testVersion` was moved into `testers.testVersion`. I rebased and updated the PR, it works now. Thank you